nixos/doc/manual/installation/installing-from-other-distro.xml

```diff
@@ -47,7 +47,7 @@
     Short version:
    </para>
 <screen>
-$ bash &lt;(curl https://nixos.org/nix/install)
+$ curl https://nixos.org/nix/install | bash
```

Comment:
Let's just do ```sh``` if we're trying to avoid bashisms.

Reply:
@matthewbauer the motivation for the PR is not to avoid bashisms, just to make the 
documentation correct even if the user 's shell is not bash.  Since the command use bash
I've assumed the install script requires bash.

Reply:
It should support generic sh. This matches the instruction on https://nixos.org/nix/.